Tests/Runtime/Attributes/BuildSceneAttributeInferredTest.cs

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) 2023-2024 Koji Hasegawa.
+// This software is released under the MIT License.
+
+using System.Threading.Tasks;
+using Cysharp.Threading.Tasks;
+using NUnit.Framework;
+using TestHelper.RuntimeInternals;
+using UnityEngine;
+
+namespace TestHelper.Attributes
+{
+    /// <seealso cref="LoadSceneAttributeTest"/>
+    [TestFixture]
+    [BuildScene]
+    public class BuildSceneAttributeInferredTest
+    {
+        private const string ObjectName = "CubeInNotInScenesInBuild";
```

Comment:
Please give it a name different from the GameObject name used in the existing BuildSceneAttributeTest.
The test will fail when run consecutively.